### PR TITLE
deprecate repo, extension

### PR DESCRIPTION
**Pull request by chrisdias**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
 	"displayName": "MongoDB",
```
Comment (EricJizbaMSFT) [inline package.json]: This looks good, but none of the changes show up in the VS Code marketplace. What if we add `[DEPRECATED]` to the 'displayName' and 'description', bump the version, and republish?

**Review by ejizba — APPROVED** [review]

**Comment by chrisdias** [comment]
The reason I initially didn't go further than this was that I didn't want to publish an updated and have anyone that does have it installed get a new version.  However, I am pretty sure we can update the .VSIX in the marketplace using the same version number which should mean that the marketplace will look right but no one will get updates.  What do you think?

**Comment by EricJizbaMSFT** [comment]
I'm really hesitant to publish a vsix with the same version. It breaks standard semver practices and I'm not sure how VS Code/the marketplace would handle it (I'm surprised they even allow you to do that).

In any case, an update is a _good_ thing IMO. It basically notifies all the users of this extension to switch over to the other extension. What's the downside to bumping the version?

**Comment by chrisdias** [comment]
ok, i pushed more changes to fully deprecate the extension.

**Comment by chrisdias** [comment]
also, let me know if you want me to publish.

**Comment by EricJizbaMSFT** [comment]
Looks good to me. Feel free to publish anytime (I don't have permissions)

**Pull request merged**